meeting_guide/wagtail_hooks.py

```diff
@@ -57,10 +55,10 @@ def queryset(self, request, queryset):
         return queryset
 
 
-class MeetingTypeAdmin(ModelAdmin):
+class MeetingTypeAdmin(SnippetViewSet):
     model = MeetingType
     menu_label = "Meeting Types"
-    menu_icon = "fa-folder-open"
+    menu_icon = "folder-open-1"

```

Comment:
The icons weren't loading for me. If memory serves, the original `fa-*` icons were from the wagtail font awesome package. I replaced these (because they weren't loading) with similar Wagtail icons that are built in.